home/views.py

```diff
@@ -140,8 +144,8 @@ def moreinfo(request, name):
     salon = get_object_or_404(Salon, url=name)
     services = Services.objects.filter(salons__name=salon.name)
     products = Products.objects.filter(salons__name=salon.name)
-    comments = Comments.objects.filter(
-        salon__id=salon.id).order_by("-created_date")
+    comments = Comments.objects.filter(salon__id=salon.id)
+    MAPS_API_KEY = os.getenv("MAPS_API_KEY")
```

Comment:
@coderjaymoh instead of importing load_dotenv in views you can simply do this.
//in settings.py
` MAPS_API_KEY = os.getenv("MAPS_API_KEY")`
//in views
` MAPS_API_KEY = settings.MAPS_API_KEY`

Reply:
Reverted to original filter by created date

Reply:
Made necesary imports on key from settings module